### PR TITLE
Simplify cql config and use IPv6 for bootstrap

### PR DESCRIPTION
**Pull request by auxten**

### CODE REVIEW

File: client/driver.go [hunk 1]
```diff
@@ -66,7 +66,8 @@ var (
 	globalSeqNo         uint64
 	randSource          = rand.New(rand.NewSource(time.Now().UnixNano()))
 
-	defaultConfigFile = "~/.cql/config.yaml"
+	// Default path of config file
```
Comment (covenantdb) [inline client/driver.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported var DefaultConfigFile should be of the form "DefaultConfigFile ..."

File: route/ipv6seed.go [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+	"github.com/pkg/errors"
+
+	"github.com/CovenantSQL/CovenantSQL/crypto"
+	"github.com/CovenantSQL/CovenantSQL/crypto/asymmetric"
+	"github.com/CovenantSQL/CovenantSQL/pow/cpuminer"
+	"github.com/CovenantSQL/CovenantSQL/proto"
+)
+
+const (
+	NODEID = "id."
```
Comment (covenantdb) [inline route/ipv6seed.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const NODEID should have comment (or a comment on this block) or be unexported

File: route/ipv6seed.go [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+	PUBKEY = "pub."
+	NONCE  = "n."
+	ADDR   = "addr."
+)
+
+type IPv6SeedClient struct{}
```
Comment (covenantdb) [inline route/ipv6seed.go:43]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type IPv6SeedClient should have comment or be unexported

File: route/ipv6seed.go [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+func (isc *IPv6SeedClient) GetBPFromDNSSeed(BPDomain string) (BPNodes IDNodeMap, err error) {
```
Comment (covenantdb) [inline route/ipv6seed.go:46]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method IPv6SeedClient.GetBPFromDNSSeed should have comment or be unexported

File: route/ipv6seed.go [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+	if err != nil {
+		return
+	}
+	addrBytes, err := crypto.RemovePKCSPadding(addrBuf)
+	if err != nil {
+		return
+	}
+
+	// NodeID
+	nodeIdBuf, err := ipv6.FromDomain(NODEID + BPDomain)
```
Comment (covenantdb) [inline route/ipv6seed.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var nodeIdBuf should be nodeIDBuf

File: route/ipv6seed.go [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+	if err != nil {
+		return
+	}
+	var nodeId proto.RawNodeID
```
Comment (covenantdb) [inline route/ipv6seed.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var nodeId should be nodeID

File: route/ipv6seed.go [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+		ID:        nodeId.ToNodeID(),
+		Addr:      string(addrBytes),
+		PublicKey: &pubKey,
+		Nonce:     *nonce,
+	}
+
+	return
+}
+
+func (isc *IPv6SeedClient) GenBPIPv6(node *proto.Node, domain string) (out string, err error) {
```
Comment (covenantdb) [inline route/ipv6seed.go:106]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method IPv6SeedClient.GenBPIPv6 should have comment or be unexported

File: route/ipv6seed.go [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+	// NodeID
+	nodeIdIps, err := ipv6.ToIPv6(node.ID.ToRawNodeID().AsBytes())
```
Comment (covenantdb) [inline route/ipv6seed.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
var nodeIdIps should be nodeIDIps

File: cmd/cql-utils/confgen.go [hunk 9]
```diff
@@ -74,13 +74,16 @@ func runConfgen() {
 	testnetConfig.PrivateKeyFile = privateKeyFileName
 	testnetConfig.PubKeyStoreFile = publicKeystoreFileName
 	testnetConfig.ThisNodeID = cliNodeID
-	testnetConfig.KnownNodes = append(testnetConfig.KnownNodes, proto.Node{
+	if testnetConfig.KnownNodes == nil {
+		testnetConfig.KnownNodes = make([]proto.Node, 1)
+	}
+	testnetConfig.KnownNodes[0] = proto.Node{
```
Comment (laodouya) [inline cmd/cql-utils/confgen.go]: If KnownNodes is not nil, this will erase first exist node.

**Review by laodouya — CHANGES_REQUESTED** [review]

File: cmd/cql-utils/confgen.go (hunk 9 above)
Comment (auxten) [reply cmd/cql-utils/confgen.go]: Fixed

**Review by laodouya — APPROVED** [review]

**Review by leventeliu — APPROVED** [review]
LGTM.

**Pull request merged**